memgpt/config.py

```diff
@@ -248,7 +250,7 @@ def __init__(
         model,
         preset=DEFAULT_PRESET,
         name=None,
-        data_sources=[],
+        data_sources=None,
```

Comment:
What's wrong with this being an empty list?

Reply:
The linter said that initialization should not use a mutating value. I think that "None" given also is different to an empty array is given (in contrast to the initialization with a default). But these are probably all just theoretical interpretations. As I wrote in the intro: I believe in making linters happy and that there is a reason why the compressed knowledge of something as the PyCharm IDE's code analyzer exists.

Reply:
It's because if you mutate the list, it becomes the default argument for all invocations of the function.

https://docs.python-guide.org/writing/gotchas/

Reply:
@vivi thanks!